adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +319,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
```

Comment:
If you run checkstyle, it will complain you the access modifier order. You should do public final instead of final public.